docs/getting-started-single-cluster.md

```diff
@@ -11,6 +11,7 @@ In this quick start, we will cover:
 - [Kind](https://kind.sigs.k8s.io/)
 - [Kubectl](https://kubernetes.io/docs/tasks/tools/)
 - [Helm](https://helm.sh/)
+- [Kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/)
```

Comment:
I recall this wasn't a pre-req as its run inside a container?
Maybe that's no longer the case?

Reply:
It looks like in the code there was some preparation for that but it doesn't see that it's been applied to all cases where kustomize is used. I can address that instead if that's preferred. 
I saw first failure at https://github.com/Kuadrant/kuadrant-operator/blob/main/hack/quickstart-setup.sh#L465
These lines would also be affected https://github.com/Kuadrant/kuadrant-operator/blob/main/hack/quickstart-setup.sh#L479-L481

Reply:
Undrestood.
Perhaps using `kubectl apply -k` would be sufficient for those cmds?